packages/designmanual/stories/pages/ExplanationService.jsx

```diff
@@ -479,6 +479,7 @@ const ExplanationService = ({ t }) => {
           selectedItem={renderSelectedItem()}
           filters={getFilters()}
           renderMarkdown={renderMarkdown}
+          totalCount={filteredItems.length}
```

Comment:
Dette er greit nok for designmanual men live vil det typisk kunne være fleire treff enn det vises på sida.